upgrading/topics/keycloak/changes.adoc

```diff
@@ -12,6 +12,17 @@ This should not affect most of the deployments, but some can be affected under s
 tried to update a user with the admin REST API and the user had some incorrect attribute changes, the update was possible. With the
 current version, the update is not possible and you will be immediately informed about the reason.
 
+==== UserModel changes
+
+The fields `username`, `email`, `firstName` and `lastName` in the `UserModel` are migrated to custom attributes as a preparation for adding more sophisticated user profiles to {project_name} in an upcoming version.
+If a database contains users with custom attributes of that exact name, the custom attributes will need to be migrated before upgrading (this is not done automatically). Otherwise, they will not be read from the database anymore and possibly deleted.
+This implies, that the `username` can now also be accessed and set via `UserModel.getFirstAttribute(UserModel.USERNAME)` (and similar for the other fields). 
```

Comment:
Change "This implies" to "This situation implies"
Change "(and similar for the other fields)." to "Similar implications exist for other fields."